tslearn/metrics/dtw_variants.py

```diff
@@ -194,7 +194,7 @@ def dtw_path(s1, s2, global_constraint=None, sakoe_chiba_radius=None,
             "One of the input time series contains only nans or has zero length.")
 
     if s1.shape[1] != s2.shape[1]:
-        raise ValueError('dimension mismatch')
+        raise ValueError("All input time series must be the same shape.")
```

Comment:
Well, they should not be the same shape (they can be of different lengths), but they should share the same number of features